core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -230,23 +230,30 @@ THE SOFTWARE.
     </div>
 
     <j:if test="${layoutType!='full-screen'}">
-    <footer>
+    <footer class="page-footer">
       <div class="container-fluid">
-        <div class="row">
-          <div class="col-md-6" id="footer"></div>
-          <div class="col-md-18">
-            <span class="page_generated">${%Page generated}: <i:formatDate value="${h.getCurrentTime()}" type="both" dateStyle="medium" timeStyle="long"/></span>
-            <j:if test="${!empty(it.api)}">
-               <span class="rest_api"><a href="api/">REST API</a></span>
-            </j:if>
-            <span class="jenkins_ver"><a href="${h.getFooterURL()}">Jenkins ver. ${h.version}</a></span>
-            <j:if test="${extensionsAvailable}">
-            <j:forEach var="pd" items="${h.pageDecorators}">
-              <st:include it="${pd}" page="footer.jelly" optional="true" />
-            </j:forEach>
-            </j:if>
+        <div class="page-footer__flex-row">
+          <div class="page-footer__footer-id-placeholder" id="footer"></div>
+
+          <j:if test="${!empty(it.api)}">
+            <div class="page-footer__links rest_api hidden-xs">
+              <a href="api/">
+                REST API<l:svgIcon href="${resURL}/images/material-icons/svg-sprite-navigation-symbol.svg#ic_arrow_forward_24px" />
+              </a>
+            </div>
+          </j:if>
+
+          <div class="page-footer__links page-footer__links--white jenkins_ver">
+            <a href="${h.getFooterURL()}">
+              Jenkins ver. ${h.version}<l:svgIcon href="${resURL}/images/material-icons/svg-sprite-navigation-symbol.svg#ic_arrow_forward_24px" />
```

Comment:
is ver needed?
```suggestion
              Jenkins v.${h.version}<l:svgIcon href="${resURL}/images/material-icons/svg-sprite-navigation-symbol.svg#ic_arrow_forward_24px" />
```

Reply:
It's what's there atm. You're proposing `v.2.222`. Wouldn't it be better `v2.222`?

Reply:
Updated, yes I think it would read better